instrumentation/opentelemetry-instrumentation-httpx/setup.cfg

```diff
@@ -48,7 +48,6 @@ install_requires =
 test =
     opentelemetry-sdk ~= 1.3
     opentelemetry-test-utils == 0.27b0
-    respx ~= 0.17.0
```

Comment:
Just curious, why is this dependency removed?

Reply:
The later versions of `httpx` are incompatible with `respx ~= 0.17.0`, so I added the different versions as dependencies`tox.ini`